src/Foundatio/Caching/CacheClientHelpers.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace Foundatio.Caching
```

Comment:
I think we already have a set of extension methods in the same file as the interface